File: data/code_pairs/fortran/93.f90
program builtinfcns
    implicit none
    real :: pi, x, y

    pi = acos(-1.0)
    x = cos(pi)
    y = sqrt(exp(log(pi)))**2

    print *, "pi = ", pi
    print *, "x = ", x
    print *, "y = ", y
end program builtinfcns

File: data/code_pairs/cpp/93.cpp
#include <iostream>
#include <cmath>

int main() {
    double pi, x, y;

    pi = acos(-1.0);
    x = cos(pi);
    y = pow(sqrt(exp(log(pi))), 2);

    std::cout << "pi = " << pi << std::endl;
    std::cout << "x = " << x << std::endl;
    std::cout << "y = " << y << std::endl;

    return 0;
}